README.md

```diff
@@ -43,6 +43,143 @@ The documentation should help configure the BinderHub service to:
+
+8. Now that we have the appropriate permissions, let's set up our configuration! Create a
+   new file named `binderhub-service-config.yaml` with the following contents:
+
+   ```yaml
+   config:
+     BinderHub:
+       use_registry: true
+       image_prefix: <repository-path>/binder
+       # Temporarily enable the binderhub UI so we can test image building and pushing
+       enable_api_only_mode: false
+   buildPodsRegistryCredentials:
+     server: "https://<region>-docker.pkg.dev"
+     username: "_json_key"
+     password: |
+       <json-key-from-service-account>
+   ```
+
+   where:
+
+   1. `<repository-path>` is what you copied from step 5.
+
+   2. `<json-key-from-service-account>` is the JSON file you downloaded in step 7.
+      This is a multi-line file - either indent it correctly to match up (the `|`
+      allows multiline strings),
+      or simply edit the contents to be a single line. Since it is JSON,
+      it does not matter.
+
+   3. `<region>` is the region your artifact repository was created in. You can see
+      this in the first part of `<repository-path>` as well.
+
+9. Run a `helm upgrade` to use the new configuration you just created:
+
+   ```bash
+   helm upgrade \
+    --install \
+    --create-namespace \
+    --devel \
+    --wait \
+    --namespace <namespace>
+    <name> \
+    binderhub-service/binderhub-service \
+    -f binderhub-service-config.yaml
+   ```
+
+   This should set up binderhub with this custom config. If you run a `kubectl -n <namespace> get pod`,
+   you will see that the binderhub pod has restarted - this confirms that the config has been set up!
+
+10. Let's verify that _image building and pushing_ works. Access the binderhub pod by following the
+    same instructions as step 3. But this time, you should see a binderhub page very similar to that
+    on [mybinder.org](https://mybinder.org). You can test build a repository here - I recommend trying
+    out `binder-examples/requirements`. It might take a while to build, but you should be able to see
+    logs in the UI. It should succeed at _pushing_ the github image, but will fail to launch. The last
+    lines in the log in the UI should look like:
+
+    ```
+    Successfully pushed europe-west10-docker.pkg.dev/binderhub-service-development/bh-service-test/binderbinder-2dexamples-2drequirements-55ab5c:50533eb470ee6c24e872043d30b2fee463d6943fBuilt image, launching...
+    Launching server...
+    Launch attempt 1 failed, retrying...
+    Launch attempt 2 failed, retrying...
+    ```
+
+    You can also go back to the Google Artifact Registry repository you created earlier to verify that the built
+    image is indeed there.
+
+11. Now that we have verified this is working, we can disable the binderhub UI as we will not be using it.
+    Remove the `config.BinderHub.enable_api_only_mode` configuration from the binderhub config, and redeploy
+    using the command from step 9.
```

Comment:
This is acceptable as it is. An alternative test can be to do:

```shell
# this is the test from the github workflow testing things as well btw
curl http://localhost:8585/build/gh/binderhub-ci-repos/cached-minimal-dockerfile/HEAD?build_only=true
```

I think I'd favor use of `curl` as it highlights this is the role of binderhub-service now - API level access to build/push to be consumed by non-binderhub provided UI.

Reply:
I'd like to keep this one as is for now, hope that's ok.

Reply:
I also spent today with Sima working on how to connect a JupyterHub to this, and I believe the GUI check 'flows' much better.